auth/server/src/main/java/com/seagates3/dao/ldap/AuthStore.java

```diff
@@ -19,6 +19,7 @@ Object find(String keyToFind, Object key, Object rootKey,
 
  public
   List<Policy> findAll(String keyToFind, Object key,
+                       Map<String, Object> parameters,
                        String prefix) throws DataAccessException;
```

Comment:
Signature change for an API is risky. Please check above comment and confirm

Reply:
This is recently written code. and I have changed all the calling instances.